src/components/TextLink.js

```diff
@@ -0,0 +1,43 @@
+import _ from 'underscore';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Text, Pressable} from 'react-native';
+import openURLInNewTab from '../libs/openURLInNewTab';
+import styles from '../styles/styles';
+
+const propTypes = {
+    /** Link to open in new tab */
+    href: PropTypes.string.isRequired,
+
+    /** Text content child */
+    children: PropTypes.string.isRequired,
+
+    /** Additional style props (optional) */
+    style: PropTypes.oneOfType([PropTypes.object, PropTypes.arrayOf(PropTypes.object)]),
+};
+
+const defaultProps = {
+    style: [],
+};
+
+const TextLink = (props) => {
```

Comment:
NABish, but I'm wondering if this component has redundant overlap with `AnchorForCommentsOnly`. Do we really need multiple implementations of links?

Reply:
There is definitely some overlap, but there is enough extra functionality that it's not easy to combine them at this point. Maybe a good cleanup issue sometime.